text/0000-symbol-name-mangling-v2.md

```diff
@@ -0,0 +1,801 @@
+- Feature Name: symbol_name_mangling_v2
+- Start Date: 2018-11-27
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a new mangling scheme that describes what the symbol names generated by the Rust compiler. This new scheme has a number of advantages over the existing one which has grown over time without a clear direction. The new scheme is consistent, does not depend on compiler internals, and the information it stores in symbol names can be decoded again which provides an improved experience for users of external tools that work with Rust symbol names. The new scheme is based on the name mangling scheme from the [Itanium C++ ABI][itanium-mangling].
```

Comment:
> describes what the symbol names generated by the Rust compiler

I think you lost a word here.

(Meta note — if you hard-wrap your text to a certain number of columns, GH comments can be more specifically targeted)

Reply:
> (Meta note — if you hard-wrap your text to a certain number of columns, GH comments can be more specifically targeted)

:+1:

Reply:
It's even better to put hard returns after each sentence and, for long sentences, each clause, but it takes some getting used to when you're writing it.